src/DDTrace/Integrations/IntegrationsLoader.php

```diff
@@ -96,8 +96,11 @@ public function __construct(array $integrations)
                 '\DDTrace\Integrations\CodeIgniter\V2\CodeIgniterSandboxedIntegration';
             $this->integrations[CurlSandboxedIntegration::NAME] =
                 '\DDTrace\Integrations\Curl\CurlSandboxedIntegration';
-            $this->integrations[ElasticSearchSandboxedIntegration::NAME] =
-                '\DDTrace\Integrations\ElasticSearch\V1\ElasticSearchSandboxedIntegration';
+            // for PHP 7.0+ use C level integration autoloader
```

Comment:
I tried putting together 5.6 version, but it caused crashes. I presume its due to bug caused by incomplete opcode handling - since we're planning migrating to zend_execute_ex for 5.x, we might want to hold off support 5.x until then.